test_scripts/API/SetAudioStreamingIndicator/ideas_SetAudioStreamingIndicator_conditions_for_SDL_to_handle_related_cases.txt

```diff
@@ -0,0 +1,79 @@
+./test_scripts/API/SetAudioStreamingIndicator/036_ATF_N_TC_SetAudioStreamingIndicator_GENERIC_ERROR_HMI_invalid_audioStreamingIndicator.lua Check_that_SDL_responds_GENERIC_ERROR_when_HMI_replies_invalid_value_audioStreamingIndicator_loop_for_wrongtype_nonexistingvalue_empty
+./test_scripts/API/SetAudioStreamingIndicator/037_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_media_app_RPC_disallowed_by_user_ExternalProprietary.lua Check_that_SDL_responds_with_USER_DISALLOWED_when_RPC_disallowed_by_user_media_app
+./test_scripts/API/SetAudioStreamingIndicator/038_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_non-media_app_RPC_disallowed_by_user_ExternalProprietary.lua Check_that_SDL_responds_with_USER_DISALLOWED_when_RPC_disallowed_by_user_non-media_app
+./test_scripts/API/SetAudioStreamingIndicator/039_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_navi_app_RPC_disallowed_by_user_ExternalProprietary.lua Check_that_SDL_responds_with_USER_DISALLOWED_when_RPC_disallowed_by_user_navi_app
+./test_scripts/API/SetAudioStreamingIndicator/040_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_comm_app_RPC_disallowed_by_user_ExternalProprietary.lua Check_that_SDL_responds_with_USER_DISALLOWED_when_RPC_disallowed_by_user_comm_app
+./test_scripts/API/SetAudioStreamingIndicator/041_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_RPC_omitted_PTgroups.lua Check_that_SDL_responds_with_DISALLOWED_when_RPC_omitted_PT_groups_media_app
+./test_scripts/API/SetAudioStreamingIndicator/042_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_RPC_omitted_PTgroups.lua Check_that_SDL_responds_with_DISALLOWED_when_RPC_omitted_PT_groups_non-media_app
+./test_scripts/API/SetAudioStreamingIndicator/043_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_RPC_omitted_PTgroups.lua Check_that_SDL_responds_with_DISALLOWED_when_RPC_omitted_PT_groups_navi_app
+./test_scripts/API/SetAudioStreamingIndicator/044_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_RPC_omitted_PTgroups.lua Check_that_SDL_responds_with_DISALLOWED_when_RPC_omitted_PT_groups_comm_app
+./test_scripts/API/SetAudioStreamingIndicator/045_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_FULL_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/046_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_LIMITED_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/047_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_BACKGROUND_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/048_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_NONE_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_app_notincluded_PT
```

Comment:
@OVikhrov, the idea is: 
allow RPC is [FULL, LIMITED, NONE] check in BACKGROUD is disallowed
allow RPC is [FULL, LIMITED, BACKGROUD] check in NONE is disallowed
allow RPC is [FULL, NONE, BACKGROUD] check in LIMITED is disallowed
allow RPC is [LIMITED, NONE, BACKGROUD] check in FULL is disallowed